client/src/controllers/ActionController.ts

```diff
@@ -46,6 +46,13 @@ export class ActionController extends Controller<
   declare continueValue: boolean;
   declare urlValue: string;
 
+  select() {
```

Comment:
```suggestion
  /**
   * Select all the text in an input or textarea element.
   */
  select() {
```

just adding some JSDoc at the top here, I can do this when merging in but it's nice to have a small description if it adds context.